docs/UserGuide.md

```diff
@@ -259,8 +259,12 @@ _Clean up Task parameters, some should be put in the feature, while things like
 ### Task parameters
```

Comment:
This shouldn't be put here. There is already a section for tasks if you scroll up.